scenario/migrate_resources_autorollback.yaml

```diff
@@ -0,0 +1,55 @@
+# This scenario is the same as migrate_resources.yaml plus
+# includes autorollback functionality at the end.
+# This scenario can be used for testing purposes and will leave
+# destination cloud at the same stage as it was before migration.
+
+namespace:
+  info_result:
+      instances: {}
+
+preparation:
+  - pre_migration_test:
+      - act_get_filter: True
+      - act_check_filter: True
+      - src_test:
```

Comment:
`check_config_quota_neutron` task is missed. Also, check if this scenario is fully up to date.